packages/tpetra/core/test/MultiVector/MultiVector_LocalViewTests.cpp

```diff
@@ -174,6 +174,8 @@ TEUCHOS_UNIT_TEST_TEMPLATE_4_DECL( MultiVector, HostView, LO, GO, Scalar , Node
   vector_t defaultVec(defaultMap);
   defaultVec.putScalar(scalar);
 
+  Kokkos::fence();
```

Comment:
Shouldn't getLocalViewHost below take care of this fence?

Reply:
You are correct.  Will fix.

Reply:
done